ci_cd.md

```diff
@@ -0,0 +1,12 @@
+### Sentry config
```

Comment:
Assume this goes along with codemagic changes? Which script in codemagic? Make sure you don't clobber the Post script we have atm for the Firebase App Dist things.

Reply:
Ah yes I should include here that it needs to be a `pre-build` step Codemagic workflow script